src/main/java/net/minecraftforge/event/entity/living/LivingTotemEvent.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.event.entity.living;
+
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.damagesource.DamageSource;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.eventbus.api.Cancelable;
+import net.minecraftforge.fml.LogicalSide;
+
+/**
+ * Fired when an Entity attempts to use a totem to prevent its death.
+ *
+ * <p>This event is {@linkplain Cancelable cancellable}, and does not {@linkplain HasResult have a result}.
+ * If this event is canceled, the totem is not used to prevent the entity's death.</p>
+ *
+ * <p>This event is fired on the {@linkplain MinecraftForge#EVENT_BUS main Forge event bus},
```

Comment:
```suggestion
 * <p>This event is fired on the {@linkplain MinecraftForge#EVENT_BUS Forge event bus},
```
Seems a bit redundant to call it "main" Forge event bus, as there's only one.

Reply:
Agreed, had copy pasted that from some other event I forget which now, will change it though.